BUILD.bazel

```diff
@@ -0,0 +1,211 @@
+load("//bazel:swift_cc_defs.bzl", "UNIT", "swift_cc_test")
+load("//:copts.bzl", "COPTS")
+
+cc_library(
+    name = "albatross",
+    hdrs = [
+        "include/albatross/Common",
+        "include/albatross/Core",
+        "include/albatross/CovarianceFunctions",
+        "include/albatross/Dataset",
+        "include/albatross/Distribution",
+        "include/albatross/Evaluation",
+        "include/albatross/ForwardDeclarations",
+        "include/albatross/GP",
+        "include/albatross/Graph",
+        "include/albatross/Indexing",
+        "include/albatross/LeastSquares",
+        "include/albatross/NullModel",
+        "include/albatross/PatchworkGP",
+        "include/albatross/Ransac",
+        "include/albatross/Samplers",
+        "include/albatross/SparseGP",
+        "include/albatross/Stats",
+        "include/albatross/Tune",
+        "include/albatross/serialize/Common",
+        "include/albatross/serialize/Core",
+        "include/albatross/serialize/Evaluation",
+        "include/albatross/serialize/GP",
+        "include/albatross/serialize/LeastSquares",
+        "include/albatross/serialize/Ransac",
+        "include/albatross/src/cereal/block_utils.hpp",
+        "include/albatross/src/cereal/dataset.hpp",
+        "include/albatross/src/cereal/distribution.hpp",
+        "include/albatross/src/cereal/eigen.hpp",
+        "include/albatross/src/cereal/evaluation.hpp",
+        "include/albatross/src/cereal/fit_model.hpp",
```

Comment:
interesting, do you have to manually add each new header for bazel?

Reply:
Yes. The way bazel works is it copies every file into a "sandbox" environment where it performs the build. The idea is to isolate the build from the host environment.

So they require that you explicitly declare all the inputs rather than just giving it include paths like cmake.